### ISSUE TITLE
video upload date and views in playlists not showing

### ISSUE THREAD

**Issue by MD77MD** [TeamNewPipe/NewPipe#3675]
#### Describe the feature you want

Video upload date and views in playlists are not showing, you must enter each video to see it's upload date or views number.
#### Is your feature request related to a problem? Please describe it

First step to sort playlist videos by date or views.
#### Additional context

This also further benefit video appearance uniformity by unifying it in all lists such as history, feed, search... .
#### How will you/everyone benefit from this feature?

This is useful when you would like to watch recent videos or videos that where created within a specific time period such as last week or in 2018.
Also watch specific videos based on views number within a playlist such as most viewed or top song

**Comment by MD77MD** [comment]
#3675
Video upload date and views in playlists not showing 

Describe the feature you want
Video upload date and views in playlists are not showing, you must enter each video to see it's upload date or views number.

Is your feature request related to a problem? Please describe it
First step to sort playlist videos by date or views.

Additional context
This also further benefit video appearance uniformity by unifying it in all lists such as history, feed, search... .

How will you/everyone benefit from this feature?
This is useful when you would like to watch recent videos or videos that where created within a specific time period such as last week or in 2018.
Also watch specific videos based on views number within a playlist such as most viewed or top song

**Comment by B0pol** [comment]
Not possible for YouTube, afaik not possible in PeenTube either, and only view count is possible in SoundCloud

**Comment by AudricV** [comment]
Update on this issue:

- YouTube, PeerTube and SoundCloud returns now upload date and view count;
- Bandcamp doesn't provide this information in the JSON of the HTML page, we need to check if that's the case with the mobile API too;
- MediaCCC: not applicable, as there is no playlist in the service.

**Comment by Vrihub** [comment]
> Update on this issue:
> 
> * YouTube, PeerTube and SoundCloud returns now upload date and view count;

Any plans about implementing this feature? At least uploaded date would be really useful.

**Comment by ghost** [comment]
Seems like in current v0.28.3 it is implemented  but only for external playlists. Not sure that local playlists need that though

**Comment by MD77MD** [comment]
<img alt="Image" src="https://github.com/user-attachments/assets/1cb4f525-9dbc-4cc5-80a2-0dc137c8cf3f" width=307 />

@dustdfg  still not working ... Neither for external or local playlists.

**Comment by ghost** [comment]
Yeah really... I need to further specify that it is currently implemented only for external playlists in grid mode

<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/d2fe74a6-ee94-452c-81aa-f91dc3af8daa" />

**Comment by MD77MD** [comment]
you are correct... so could this be added to list view as well... i don't imagine its hard given its already implemented in grid view
